docs/api.md

```diff
@@ -324,6 +324,8 @@ When passing an EventTarget, you can provide `useCapture` as the 3rd parameter,
 
 When the stream ends (for example, by using [take](#take), [takeUntil](#until), etc.), it will automatically be disconnected from the event source.  For example, in the case of DOM events, the underlying DOM event listener will be removed automatically.
 
+When source event has more than one argument, all the arguments will be agregated into array in resulting Stream.
+
```

Comment:
How about this: Since this array behavior is only supported for EventEmitter (EventTarget is always single-valued), we could refactor the "Note on EventEmitter" section below to have 2 notes.  What do you think of something like this:

**Notes on EventEmitter**

1. When source event has more than one argument ...
2. EventEmitter allows events to be delivered in the same tick ...